spec/features/burgers_features/burger_add_spec.rb

```diff
@@ -1,17 +1,33 @@
 require "rails_helper"
 
 feature "User visits new burger page" do
+  let(:restaurant) { Restaurant.create(name: "restaurant", location: "Boston", dining_type: "Might be a selction", description: "Really long lines and really long text", hours: "10am-10pm") }
+  let(:restaurant) { FactoryGirl.create(:restaurant) }
   scenario "User sees a form to enter burger information" do
```

Comment:
Please use let!() syntax instead of lazy let(). A quick google will give context